File: Beem.Server/ClientApp/src/app/models/news.ts
export class News {
    constructor(init?:Partial<News>){
        Object.assign(this, init);
    }
    Id: number = 0
    Title: string = "";
    Img: string = "";
    ImgHeader: string = "";
    NewsText: string = "";
    PublicationDateTime: Date = new Date();
    IsActive: boolean = false;
}

File: Beem.Server/Models/News.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace Beem.Server.Models
{
    [Index(nameof(PublicationDateTime))]
    public class News
    {
        [Key]
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string NewsText { get; set; } = string.Empty;
        public string? Img { get; set; }
        public string? ImgHeader { get; set; }
        
        public DateTime PublicationDateTime { get; set; }
        public bool IsActive { get; set; }
    }
}